lib/msal-node/src/cache/serializer/EntitySerializer.ts

```diff
@@ -12,10 +13,11 @@ export class EntitySerializer {
       * @param key
       */
     static mapAccountKeys(accCache: AccountCache, key: string): object {
```

Comment:
Some extra stuff in this file that I think is no longer needed (commented out)

Reply:
+1 for removing unneeded comments

Reply:
This is being fixed by @sameerag.